package.json

```diff
@@ -1,39 +1,44 @@
 {
-  "name": "goals-site",
+  "name": "goal-site",
   "version": "1.0.0",
   "private": true,
   "scripts": {
     "dev": "npm-run-all --parallel dev:*",
     "dev:nuxt": "nuxt -p 3443",
     "dev:https-link": "local-ssl-proxy --source 443 --target 3443",
+    "serve": "nuxt",
     "build": "nuxt build",
-    "start": "nuxt start -p 3001",
+    "start": "nuxt start",
     "generate": "nuxt generate",
     "lint:js": "eslint --ext \".js,.vue\" --ignore-path .gitignore .",
-    "lint": "yarn lint:js"
+    "lint": "yarn lint:js",
+    "lintfix": "yarn lint:js --fix",
```

Comment:
tried running the lint fix command, Found this error
![image](https://user-images.githubusercontent.com/57758447/185787521-513ff544-100e-4843-a6a9-435a12871171.png)